recipes/flytekitplugins-athena/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "flytekitplugins-athena" %}
+{% set version = "1.0.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flytekitplugins-athena-{{ version }}.tar.gz
+  sha256: 7cb730685cfe9028e2a4b9b523d418c5255cf54a82173868873359408f53d7a0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - flytekit >=1.0.0b3,<1.2.0
+
+test:
+  imports:
+    - flytekitplugins.athena
+  # commands:
+  #   - pip check
```

Comment:
Can you just quickly say why you commented this out? Then this should be ready to go.

Reply:
The pip checks are commented out only when after the builds pass but the pip-checks were failing.

Reply:
Ok but why were they commented out? Is that something that cannot be fixed?

Reply:
As far as I recall, that was the case. It was complaining about not finding the same package that was already installed.

Reply:
Ok strange. Maybe investigate this in the feedstock?